draft-ietf-quic-qpack.md

```diff
@@ -580,40 +581,48 @@ prefix integer.  The remainder of the string literal is unmodified.
 A string literal without a prefix length noted is an 8-bit prefix string literal
 and follows the definitions in [RFC7541] without modification.
 
-## Stream Types
+## Instructions
 
-QPACK instructions occur in three locations, each of which uses a separate
-instruction space:
+There are three separate QPACK instruction spaces. Encoder instructions
+({{encoder-instructions}}) carry table updates, decoder instructions
+({{decoder-instructions}}) carry acknowledgments of table modifications and
+header processing, and header block instructions ({{header-block-instructions}})
+compress a header list by referring to the QPACK table state.
```

Comment:
Well, they don't really *compress* the header list, they are the compression output.  Perhaps "reconstruct" a header list?

Reply:
Well, an encoder view of the header block is not to reconstruct the list. 

The initial text simply stated that these instructions refer to tables. Adding flavour seems to complicate the matter, unless we think spelling out verbatim is useful. 

Would something like "header block instructions convey an encoded representation of a header list by referring to the QPACK table state" suffice?

Reply:
WFM.